BreakingChanges.md

```diff
@@ -0,0 +1,44 @@
+## Breaking changes in 2.0.0
```

Comment:
Yeah, I am getting more details by migrating our sumo code base. Will enrich the doc with these.